data/modules/Equipment/Internal.lua

```diff
@@ -0,0 +1,242 @@
+	l10n_key="RADAR",
+	price=680, purchasable=true, tech_level=3,
+	slot = { type="sensor.radar", size=1 },
+	mass=1.0, volume=1.0, capabilities = { radar=1 },
+	icon_name="equip_radar"
+})
+
+--===============================================
+-- Shield Generators
+--===============================================
+
+Equipment.Register("shield.basic_s1", EquipType.New {
+	l10n_key="SHIELD_GENERATOR",
+	price=2500, purchasable=true, tech_level=5,
+	slot = { type="shield", size=1 },
+	mass=1, volume=2, capabilities = { shield=1 },
+	icon_name="equip_shield_generator"
+})
+
+Equipment.Register("shield.basic_s2", EquipType.New {
+	l10n_key="SHIELD_GENERATOR",
+	price=5500, purchasable=true, tech_level=7,
+	slot = { type="shield", size=2 },
+	mass=2.8, volume=5, capabilities = { shield=2 },
+	icon_name="equip_shield_generator"
+})
+
+Equipment.Register("shield.basic_s3", EquipType.New {
+	l10n_key="SHIELD_GENERATOR",
+	price=11500, purchasable=true, tech_level=8,
+	slot = { type="shield", size=3 },
+	mass=7, volume=12.5, capabilities = { shield=3 },
+	icon_name="equip_shield_generator"
+})
+
+Equipment.Register("shield.basic_s4", EquipType.New {
+	l10n_key="SHIELD_GENERATOR",
+	price=23500, purchasable=true, tech_level=9,
+	slot = { type="shield", size=4 },
+	mass=17.9, volume=32, capabilities = { shield=4 },
+	icon_name="equip_shield_generator"
+})
+
+Equipment.Register("shield.basic_s5", EquipType.New {
+	l10n_key="SHIELD_GENERATOR",
+	price=58500, purchasable=true, tech_level=10,
+	slot = { type="shield", size=5 },
+	mass=43.7, volume=78, capabilities = { shield=5 },
+	icon_name="equip_shield_generator"
+})
+
+--===============================================
+-- Hull Modifications
+--===============================================
+
+Equipment.Register("hull.atmospheric_shielding", EquipType.New {
+	l10n_key="ATMOSPHERIC_SHIELDING",
+	price=200, purchasable=true, tech_level=3,
+	slot = { type="hull.atmo_shield", size=1 },
+	mass=1, volume=2, capabilities = { atmo_shield=9 },
+	icon_name="equip_atmo_shield_generator"
+})
+
+Equipment.Register("hull.heavy_atmospheric_shielding", EquipType.New {
+	l10n_key="ATMOSPHERIC_SHIELDING_HEAVY",
+	price=900, purchasable=true, tech_level=5,
+	slot = { type="hull.atmo_shield", size=3 },
+	mass=5, volume=12, capabilities = { atmo_shield=19 },
+	icon_name="equip_atmo_shield_generator"
+})
+
+Equipment.Register("misc.hull_autorepair", EquipType.New {
+	l10n_key="HULL_AUTOREPAIR", slots="hull_autorepair",
+	price=16000, purchasable=true, tech_level="MILITARY",
+	slot = { type="hull.autorepair", size=4 },
+	mass=30, volume=40, capabilities={ hull_autorepair=1 },
+	icon_name="repairs"
+})
+
+--===============================================
+-- Thruster Mods
+--===============================================
+
+Equipment.Register("misc.thrusters_default", ThrusterType.New {
+	l10n_key="THRUSTERS_DEFAULT", slots="thruster",
+	price=120, purchasable=true, tech_level=2,
+	slot = { type="thruster", size=1 },
```

Comment:
Need thrusters of varying sizes; currently only ships which require S1 thrusters can fit thrusters..

Reply:
#5950